### DISCUSSION TITLE
enable gitter.im, bridge to libera IRC and add a gitter sidecar to docs

### DISCUSSION THREAD

**Discussion by graingert** [Textualize/textual#80] [Ideas]
the first step is to enable the community at https://gitter.im/willmcgugan/#createcommunity then create the room at https://gitter.im/willmcgugan/textual

then you'll need to register a Libera room with ChanServ https://libera.chat/guides/creatingchannels

then you'll need to grab the matrix room ID of the libera room (you can see it when you connect via https://matrix.to/#/#textual:libera.chat ) and send an email like:

```mime
MIME-Version: 1.0
Date: Fri, 13 Aug 2021 14:43:53 +0100
Message-ID: <redacted@mail.gmail.com>
Subject: merging the sqlalchemy-redshift and Libera and gitter chat rooms
From: Thomas Grainger <redacted@gmail.com>
To: support@gitter.im
Content-Type: text/plain; charset="UTF-8"

please could you merge !jhiGysNVVNcUzpDOeW:libera.chat with
https://gitter.im/sqlalchemy-redshift/sqlalchemy-redshift

thanks

Thomas Grainger
```

then for the docs integration you can grab the JS snip from https://sidecar.gitter.im/

**Comment by graingert** [comment]
gitter room is now at https://gitter.im/textual-ui/community
libera room is at https://matrix.to/#/#textual-ui:libera.chat AKA https://matrix.to/#/!IPpLNXgkNulLBkfjBT:libera.chat?via=libera.chat&via=matrix.org

**Comment by tusharsadhwani** [comment]
Can you connect the matrix tg bridge to the matrix chat?
Instructions available here: https://t2bot.io/telegram/

I'd like to be able to chat via telegram, so I've made a group for it: https://t.me/textual_ui

**Reply by tusharsadhwani** [reply]
The group's id is -1001611551039
